caravel/viz.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, datasource, form_data, slice_=None):
         if isinstance(form_data, (MultiDict, ImmutableMultiDict)):
             form = form_class(form_data)
         else:
-            form = form_class(**form_data)
+            form = form_class(MultiDict(form_data))
```

Comment:
how about:
```
if not isinstance(form_data, (MultiDict, ImmutableMultiDict)):
    form_data = MultiDict(form_data)
form = form_class(form_data)
```
that way form_data becomes a bit more uniform from that point onward